### PR TITLE
Amended previous version as 0.10.1 with updates to release notes

### PR DESCRIPTION
**Pull request by cmoesgaard**
- [x] Have you updated the documentation?
- [x] Have you performed a smoke test of the application?
- [x] Have you written tests for your changes?
- [x] Have you updated the release notes?

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -12,6 +12,11 @@ New features
   ``organisationfunktion``, we can now represent the account name.
 * #22849: Added handling for user permissions, giving a fitting error if a user attempts an action without the correct permissions.
 * #23976: Employees with their associated relations can now be created with one API call. All requests are now validated before being submitted to LoRa, to prevent half-writes.
+* #24134: Columns in the UI can now be sorted. There is currently a known bug where sorting doesn't work after performing an update.
```
Comment (danchr) [inline NEWS.rst]: We should have a separate section for known bugs IMO.

File: NEWS.rst [hunk 2]
```diff
@@ -12,6 +12,11 @@ New features
...
+* #24135: Dropdowns are now automatically sorted.
```
Comment (danchr) [inline NEWS.rst]: `s/automatically/alphabetically/`

File: NEWS.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Version 0.10.0, 2018-10-07
+Version 0.10.1, 2018-10-08
```
Comment (danchr) [inline NEWS.rst]: Erm, weren't we going to simply overwrite `0.10.0`?

Comment (cmoesgaard) [reply NEWS.rst]: It's been tagged already. Rewriting published tags is messy business.

**Pull request merged**